pkg/backup/backup_test.go

```diff
@@ -9467,13 +9448,7 @@ func TestExportRequestBelowGCThresholdOnDataExcludedFromBackup(t *testing.T) {
 	ctx := context.Background()
 	localExternalDir, cleanup := testutils.TempDir(t)
 	defer cleanup()
-	args := base.TestClusterArgs{
-		ServerArgs: base.TestServerArgs{
-			// Test fails when run within a tenant as zone config
-			// updates are not allowed by default. Tracked with 73768.
-			DefaultTestTenant: base.TODOTestTenantDisabled,
-		},
-	}
+	args := base.TestClusterArgs{ServerArgs: base.TestServerArgs{}}
```

Comment:
There was zero to almost no effort in enabling tenant testing for most of them, but not that straightforward to do that for `TestExportRequestBelowGCThresholdOnDataExcludedFromBackup ` and `TestExcludeDataFromBackupDoesNotHoldupGC`. They fail because backpressure doesn't build up in `upsertUntilBackpressure`. Steven and I took a high level look and maybe the reason it doesn't build-up here is secondary tenant's spans are not included in backpressure checks: https://github.com/cockroachdb/cockroach/blob/e048c1b1f3821a7ac57c6c8ca2077084150849fa/pkg/kv/kvserver/replica_backpressure.go#L86-L94